packages/babel-parser/test/helpers/to-fuzzed-options.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-confusing-arrow */
+const random = (min, max) => Math.floor(Math.random() * (max - min + 1) + min);
+const clone = value => JSON.parse(JSON.stringify(value));
+
+const { TEST_FUZZ } = process.env;
+
+const toDescriptorAssignedObject = (delta, object) =>
+  delta.reduce(
+    (object, [key, descriptor]) => (
+      !descriptor || descriptor.delete
+        ? delete object[key]
+        : Object.defineProperty(object, key, descriptor),
+      object
+    ),
+    clone(object),
+  );
+
+const toAdjustFunction = adjustments =>
+  !adjustments || Object.keys(adjustments).length === 0
+    ? null
+    : Object.assign(
+        (adjust, value, key, parent) =>
+          key in adjustments
+            ? adjustments[key](adjust, value, key, parent)
+            : value,
+        adjustments,
+      );
+
+const SyntaxErrorMessageRegExp = /\((\d+):(\d+)\)$/;
+const toAdjustedSyntaxError = (adjust, error) =>
+  error && SyntaxErrorMessageRegExp.test(error.message)
+    ? SyntaxError(
+        error.message.replace(/\((\d+):(\d+)\)$/, function (_, line, column) {
+          const loc = {
+            line: parseInt(line, 10),
+            column: parseInt(column, 10),
+          };
+          return `(${adjust(
+            adjust,
+            loc.line,
+            "line",
+            loc,
+          )}:${adjust(adjust, loc.column, "column", loc)})`;
+        }),
+      )
+    : error;
+
+export default function toFuzzedOptions(options) {
+  if (TEST_FUZZ === "false") return [[false, options]];
```

Comment:
Can we make `TEST_FUZZ` opt-in? As long as CI will run fuzz testing, we don't have to run them in local environments.

Reply:
I'm fine with that. I think I might want to push back on that later when (hopefully) more of the options are tested through fuzz testing (like the errorRecovery I gave above), especially since the entirety of the testing time is dominated by file-reads vs. actual "testing". However, as this only tests the line number stuff right now, I'm fine with reversing this and turning it on in CI.

Reply:
Done: https://github.com/babel/babel/pull/14201/commits/b1118e410f39e2f73ade31720e42485de7cf793a